frontend/src/pages/joinToLobby/JoinToLobby.js

```diff
@@ -63,19 +63,19 @@ export const JoinToLobbyPage = () => {
             />
           )}
 
-          <Layout pt3>
-            <LayoutedLobby {...lobbyDetailState.data} />
-          </Layout>
-
-          <Layout flex justify-center pt3>
-            <Button
-              primary
-              disabled={joinToLobbyState.isLoading}
-              onClick={onJoinToLobbyMemoized}
-            >
-              {t('Page.JoinToLobby.JoinButton')}
-            </Button>
-          </Layout>
+          <LayoutedLobby
+            {...lobbyDetailState.data}
+            JoinToLobby={
```

Comment:
prosp s malym pocatecnim pismenem prosim a spis bych to pojemnoval jako footer, je to vic pouzitelnejsi, kdyby se dana komponenta pouzivala na dalsim miste